header.go

```diff
@@ -513,6 +514,85 @@ func fixUnquotedSpecials(s string) string {
 	return clean.String()
 }
 
+func fixUnescapedQuotes(s string) string {
+	split := strings.SplitAfter(s, ";")
+	fresh := &strings.Builder{}
```

Comment:
```suggestion
	sb:= &strings.Builder{}
```

for String Builder; although `fresh` did make me smile.